MSGraphSDKNXOAuth2/Auth/NXOAuth2AuthenticationProvider.m

```diff
@@ -188,9 +188,17 @@ - (void)loginWithViewController:(UIViewController*)viewController
     }];
 }
 
-//- (BOOL)loginSilent {
-//    // Placeholder: Load an existing account from the account store
-//}
+- (BOOL)loginSilent {
+    // Placeholder: Load an existing account from the account store
+    int accountCount = [[NXOAuth2AccountStore sharedStore] accounts].count;
+    
+    if (accountCount == 0) {
+        return NO;
+    } else {
+        self.userAccount = [[NXOAuth2AccountStore sharedStore] accounts][accountCount - 1];
```

Comment:
Aight, I could throw in a user preference to hold the identifier and get the very last one.
Will do another pull request shortly, let me know what you think about that approach.